backend/api/network.py

```
@@ -36,14 +36,26 @@ def extract_emails(text: str | None) -> list[str]:
 
 @router.get("/graph", response_model=GraphResponse)
 async def get_network_graph(
-    limit: int = 500, user_id: str | None = None, db: AsyncSession = Depends(get_db), current_user: str = Depends(get_current_user)
+    limit: int = 500,
+    user_id: str | None = None,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)